src/ol_dbt/models/intermediate/micromasters/_int_micromasters__models.yml

```
@@ -2,6 +2,56 @@
 version: 2
 
 models:
+- name: int__micromasters__dedp_proctored_exam_grades
+  description: DEDP proctor exam grades stored in MicroMasters database. We haven't
+    had any new data for this table since 2022-09.
+  columns:
+  - name: program_title
+    description: str, title of the program
+    tests:
+    - not_null
+  - name: micromasters_program_id
+    description: int, id of the program in the micromasters database
+  - name: mitxonline_program_id
+    description: int, id of the program in the mitxonline database
+  - name: course_title
+    description: str, title of the course for example Foundations of Modern Finance
+      I
+  - name: course_number
+    description: str, edX course number for example CTL.SC4x or 15.415.1x
```

since this table only applies to DEDP, we might as well use DEDP examples. 

```suggestion
    description: str, edX course number for example 14.73x or JPAL102x
```